build-pipeline/README.md

```diff
@@ -0,0 +1 @@
+The contents of the [build-pipeline](build-pipeline) folder are used by the .NET engineering infrastructure to build and publish the Docker images to Docker Hub.
```

Comment:
Consider adding a link to https://hub.docker.com/r/microsoft/dotnet-framework/

Reply:
Added link to both repos - https://hub.docker.com/r/microsoft/dotnet-framework/ and https://hub.docker.com/r/microsoft/dotnet-framework-build/